contracts/EasyAuction.sol

```diff
@@ -135,6 +135,13 @@ contract EasyAuction is Ownable {
         feeNumerator = newFeeNumerator;
     }
 
+    // @dev: function to intiate a new auction
+    // Warning: In case the auction is expected to raise more than
+    // 2^96 units of the biddingToken, don't start the auction, as
+    // it will not be settlable. E.g, one can not raise more than
+    // 7.9e10 DAI.
+    // Also make sure that prices between biddingToken and auctioningToken
+    // are well presented by a fraction of uint96/uint96
```

Comment:
This separates the notice from the warning above and makes it more like a dev comment than an actual issue with the code.

```suggestion
    //
    // Prices between biddingToken and auctioningToken are expressed by a
    // fraction whose components are stored as uint96.
```